index.html

```diff
@@ -9,7 +9,7 @@
 		<title>MyTimetable</title>
 		<meta
 			name="description"
-			content="A drag-and-drop timetable planner for University of Adelaide students."
+			content="MyTimetable is a simple drag-and-drop timetable planner for University of Adelaide students."
```

Comment:
The meta description should be about 150-220 characters, so maybe something like "MyTimetable is a simple drag-and-drop timetable planner for University of Adelaide students. Easily organise your course classes and create the perfect timetable with this tool by the CS Club Open Source Team."